pokemongo_bot/cell_workers/find_biggest_cluster.py

```diff
@@ -0,0 +1,40 @@
+from pokemongo_bot.cell_workers.utils import distance, merc2coord, coord2merc
+from networkx.algorithms.clique import find_cliques
+
+import networkx as nx
+import numpy as np
+
+
+class FindBiggestCluster(object):
```

Comment:
On another note. The original idea was to seperate the cluster-finder because it can find clusters of any kind not just forts. So for future functionality it could perhaps find the biggest pokets of pokemons for example. Thats why perhaps it would be a good idea to be seperate, so other tasks could use it? I could put it into utils.